FirstTask.py

```diff
@@ -0,0 +1,219 @@
+from os import system as sys
+
+logout_check = False
+
+class BankAccount:
+    def __init__(self, name, password, account_no, balance, account_type):
+        self.name = name
+        self.balance = balance
+        self.account_no = account_no
+        self.account_type = account_type
+        self.password = password
+
+    def account_number(self):
+        return self.account_no
+
+    def pass_code(self):
+        return self.password
+    
+    def check_balance(self):
+        return self.balance
+
+    def check_account_type(self):
+        return self.account_type
+    
+    def deposit_bill(self, bill):
+        if bill >= 0:
+            self.balance += bill
+            return True
+        else:
+            return False
+    
+    def draw_money(self, money):
+        if self.balance >= money:
+            self.balance -= money
+            return True
+        else:
+            return False
+    
+    def account_details(self):
+        print("Account # ", self.account_no)
+        print("Name: ", self.name)
+        print("Balance: ", self.balance)
+        print("Account Type: ", self.account_type)
+  
+class Bank:
+
+    account_list = []
+
+    def create_account(self, account):
+        self.account_list.append(account)
+    
+    def login(self, account_number, code):
+        user1=self.find_account(account_number)

```

Comment:
You can unpack multiple values returned by a method in python. So, it can be done like this:
```suggestion
        status, response = self.find_account(account_number)
```
Where `status` is always a boolean value and `response` would be an account object or a message depending upon the `status` value.